src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlInternalConnectionTds.cs

```diff
@@ -2795,7 +2802,7 @@ internal SqlFedAuthToken GetFedAuthToken(SqlFedAuthInfo fedAuthInfo)
             // Username to use in error messages.
             string username = null;
 
-            var authProvider = _sqlAuthenticationProviderManager.GetProvider(ConnectionOptions.Authentication);
+            SqlAuthenticationProvider authProvider = SqlAuthenticationProvider.GetProvider(ConnectionOptions.Authentication);
```

Comment:
With this change, you can now remove the local variable _sqlAuthenticationProviderManager (lines 142 and 499). This change in the netcore code was to eliminate loading of the manager if AAD auth isn't even used and this syncs it to netfx. Even though technically unrelated to the PR description, my thought was to try to sync netfx and netcore and bit more with future code merging in mind.

Reply:
I noticed that. I also made a few other changes to align .NET Core and .NET Framework more closely. Should I create a separate PR for those changes, or are you okay with including them in this one?

Reply:
I'd just include it here.

Reply:
done